Source/WebCore/platform/graphics/GeometryUtilities.cpp

```diff
@@ -346,4 +346,10 @@ float angleOfPointToSideOfIntersection(const FloatRect& boundingRect, const std:
     return side == BoxSide::Top || side == BoxSide::Bottom ? angle : 90 - angle;
 }
 
+float normalizeAngle(float radians)
+{
+    double circles = radians / radiansPerTurnDouble;
+    return radiansPerTurnDouble * (circles - floor(circles));
+}
```

Comment:
The input and the input and the output of this function are float. But the calculation inside it is in double precision. Is there a reason for not doing it in float and use radiansPerTurnFloat instead? Does this make the calculation more accurate?

Reply:
Seems like we should use `radiansPerTurnFloat` here?